client/src/pages/OfferHelp.js

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from "react";
 import { withRouter, Link } from "react-router-dom";
-
+import {Toast } from "antd-mobile"; //263
```

Comment:
- Let's replace this line with `import { Toast } from "antd-mobile";`
- Let's remove all comments